eth/backend.go

```diff
@@ -250,6 +255,10 @@ func New(stack *node.Node, config *ethconfig.Config) (*Ethereum, error) {
 		return nil, err
 	}
 
+	if config.TriesVerifyMode.NeedRemoteVerify() {
```

Comment:
I suggest we always have a `VerifyManager`, for local verification and unsecure verification, it is an dummy implementation.